### PR TITLE
Eliminate LaTeX-style quotations from compiler output

### PR DESCRIPTION
**Pull request by thestuckster**
Fixes Latex formatting issues due to trailing '.

### CODE REVIEW

**Comment by thestuckster** [comment]
I have signed the CLA

**Comment by retronym** [comment]
Many of these changes affect the content of compiler error messages, which is why [some tests are failing](https://scala-ci.typesafe.com/job/scala-2.13.x-validate-main/377/testReport/). You would need to run `sbt 'partest --update-check'` to automatically update the `test/files/*.check` files for the failing tests.

**Comment by thestuckster** [comment]
@retronym thanks for the comment, that will help a lot! I thought I was going to have to manually update them all and I just haven't had the time do it so that is a godsend

File: src/build/dbuild-meta-json-gen.scala [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 import com.typesafe.dbuild.model._
 
+
```
Comment (SethTisue) [inline src/build/dbuild-meta-json-gen.scala:15]: this change appears to been inadvertent

**Review by SethTisue — CHANGES_REQUESTED** [review]
looks good except for the one accidentally modified file. also, before final merge, can you squash all of this into a single commit, please?

**Comment by SethTisue** [comment]
Squashing to a single commit should un-confuse Scabot & Jenkins, I think.

**Comment by thestuckster** [comment]
Can do, I'll revert the change on that file and I'll make a new branch / PR with just one commit for all the changes to make it easier on the build system

**Comment by SethTisue** [comment]
note that a new PR isn't necessary; you can force-push to this one. (but if you find it easier to just make a new PR instead, that's fine too.)

**Comment by thestuckster** [comment]
Making a new PR seemed the easiest option IMO, you can find the new one here: https://github.com/scala/scala/pull/6418

thanks for the help!

**Pull request closed without merging**